features/other_a/abuse_report.feature

```diff
@@ -6,107 +6,59 @@ Feature: Filing an abuse report
   Scenario: File an abuse request with default options
 
   Given I have a work "Illegal thing"
```

Comment:
Are we actually using this work in this test? It looks to me like we could remove this step and save a tiny tiny bit of time.